dev/com.ibm.ws.security.oidc.client_fat/publish/servers/com.ibm.ws.security.openidconnect.client-1.0_fat.rp/configs/rp_server_withOpStub_encrypt.xml

```diff
@@ -0,0 +1,1097 @@
+		authFilterRef="authFilterRS256"
+		signatureAlgorithm="RS256"
+		trustStoreRef="trust_allSigAlg"
+		trustAliasName="rs256"
+	>
+	</openidConnectClient>	
+	
+	<authFilter id="authFilterRS384">
+		<requestUrl
+			id="requestUrlRS384"
+			urlPattern="/simple/RS384"
+			matchType="contains" />
+	</authFilter>
+
+	<openidConnectClient
+		id="rs384"
+		scope="openid profile"
+		clientId="client01"
+		clientSecret="{xor}LDo8LTor"
+		mapIdentityToRegistryUser="true"
+		httpsRequired="false"
+		redirectToRPHostAndPort="http://localhost:${bvt.prop.security_2_HTTP_default}"
+		authorizationEndpointUrl="http://localhost:${bvt.prop.security_1_HTTP_default}/oidc/endpoint/OidcConfigSample/authorize"
+		tokenEndpointUrl="http://localhost:${bvt.prop.security_1_HTTP_default}/TokenEndpointServlet/getToken"  
+		validateAccessTokenLocally="true"
+		authFilterRef="authFilterRS384"
+		signatureAlgorithm="RS384"
+		trustStoreRef="trust_allSigAlg"
+		trustAliasName="rs384"
+	>
+	</openidConnectClient>	
+	
+	<authFilter id="authFilterRS512">
+		<requestUrl
+			id="requestUrlRS512"
+			urlPattern="/simple/RS512"
+			matchType="contains" />
+	</authFilter>
+
+	<openidConnectClient
+		id="rs512"
+		scope="openid profile"
+		clientId="client01"
+		clientSecret="{xor}LDo8LTor"
+		mapIdentityToRegistryUser="true"
+		httpsRequired="false"
+		redirectToRPHostAndPort="http://localhost:${bvt.prop.security_2_HTTP_default}"
+		authorizationEndpointUrl="http://localhost:${bvt.prop.security_1_HTTP_default}/oidc/endpoint/OidcConfigSample/authorize"
+		tokenEndpointUrl="http://localhost:${bvt.prop.security_1_HTTP_default}/TokenEndpointServlet/getToken"  
+		validateAccessTokenLocally="true"
+		authFilterRef="authFilterRS512"
+		signatureAlgorithm="RS512"
+		trustStoreRef="trust_allSigAlg"
+		trustAliasName="rs512"
+	>
+	</openidConnectClient>	
+-->	
+	
+<!--  diff sign and encrypt varieties -->	
+
+	<authFilter id="authFilterHS256RS256">
+		<requestUrl
+			id="requestUrlHS256RS256"
+			urlPattern="/simple/SignHS256EncryptRS256"
+			matchType="contains" />
+	</authFilter>
+
+	<openidConnectClient
+		id="SignHS256EncryptRS256"
+		scope="openid profile"
+		clientId="client01"
+		clientSecret="{xor}LDo8LTor"
+		sharedKey="secret"
+		mapIdentityToRegistryUser="true"
+		httpsRequired="false"
+		redirectToRPHostAndPort="http://localhost:${bvt.prop.security_2_HTTP_default}"
+		authorizationEndpointUrl="http://localhost:${bvt.prop.security_1_HTTP_default}/oidc/endpoint/OidcConfigSample/authorize"
+		tokenEndpointUrl="http://localhost:${bvt.prop.security_1_HTTP_default}/TokenEndpointServlet/getToken"  
+		validateAccessTokenLocally="true"
+		authFilterRef="authFilterHS256RS256"
+		keyManagmetnKeyAlias="rs256"
+		sslRef="ssl_allSigAlg"
+	>
+	</openidConnectClient>
+
+	<authFilter id="authFilterHS384RS256">
+		<requestUrl
+			id="requestUrlHS384RS256"
+			urlPattern="/simple/SignHS384EncryptRS256"
+			matchType="contains" />
+	</authFilter>
+
+	<openidConnectClient
+		id="SignHS384EncryptRS256"
```

Comment:
This and the SignHS512 config will need to have `signatureAlgorithm="HS384"` (and `"HS512"`) added. (That's assuming we do decide to add those HS options when we're updating our algorithm support.)

Reply:
I didn't spend much time on new HS algorithms - the issue we have for new signature algs focused on ES and RS - wasn't spending much time on HS until I knew if we were going to support them
Will update as appropriate when I get the runtime code